distribution/sender/routes/db-sync-route.xml

```diff
@@ -4,100 +4,68 @@
         <from uri="direct:out-bound-db-sync" />
 
         <log message="Start db-sync-route: ${body}" loggingLevel="DEBUG" />
-
+        
+        <setProperty name="isValid">
```

Comment:
Because there's a chance that no entity for the uuid exists. The current(HEAD) route have:

When no entity was found, a **null** value is written in the **body**.
 ```
           <otherwise>
                <log message="Loading entity from DB..." loggingLevel="DEBUG" />

                <toD uri="openmrs:extract?tableToSync=${exchangeProperty.event.tableName.toUpperCase()}&amp;uuid=${exchangeProperty.event.identifier}" />

                <log loggingLevel="DEBUG" message="Loaded entity -> ${body}" />

                <choice>
                    <when>
                        <simple>${body.size()} == 1</simple>
                        <setBody>
                            <jsonpath>$[0]</jsonpath>
                        </setBody>
                    </when>
                    <otherwise>
                        <setBody>
                            <simple>${null}</simple>
                        </setBody>
                    </otherwise>
                </choice>
            </otherwise>
```

Then, if the body is null **and** the operation is <> 'r', we discard this (see the bellow log)
```
        <choice>
            <when>
                <simple>${body} != null || ${exchangeProperty.event.operation} == 'r'</simple>
                ...
                <log message="Entity sent!" />
            </when>
            <otherwise>
                <!-- TODO Log to a special failures log file or DB -->
                <log message="No entity found in the database matching identifier: ${exchangeProperty.event.identifier}" loggingLevel="ERROR" />
            </otherwise>
        </choice>
```

So, the above property was created to maintain this behavior, and to avoid creating `SenderSyncMessage` entities that doesn't have a corresponding database register for the uuid (`operation <> 'r'`), and by that, does not have conditions to be sent to the central.

If we create a `SenderSyncMessage` for every request, even for those without corresponding database register for the uuid (`operation <> 'r'`), we will need to delete the `SenderSyncMessage` to avoid infinite reprocessing (in the `active-mq-publisher` route) when no database row exists for the `SenderSyncMessage` uuid.